website/photos/forms.py

```diff
@@ -18,7 +45,7 @@ def __init__(self, *args, **kwargs):
         if "instance" in kwargs and "_cover" in self.fields:
             self.fields["_cover"].queryset = Photo.objects.filter(album=self.instance)
 
-    album_archive = forms.FileField(
+    album_archive = FilePondField(
         required=False,
```

Comment:
Does this actually need a FilePondField? Or would a FileField with a FilePondWidget suffice?

Reply:
The FilePondField needs to extract the file either from the upload_id in the data or from the uploaded files in the multipart/formdata. This is because without javascript this falls back to a regular file input